pages/common/pop.md

```diff
@@ -0,0 +1,16 @@
+# pop
+
+> Send emails from your terminal.
+> More information: <https://github.com/charmbracelet/pop>.
+
+- Launch the Text-based User Interface:
+
+`pop`
+
+- Send email with file content as body:
```

Comment:
Maybe ?
```suggestion
- Send an email with an attached file:
```

Reply:
Something like "Populate Body/Content with specified file"?

Reply:
`Use file content to populate the email's body`

@mfrw: Perhaps that would be better?

Also, I'm not sure if the different sections of the email deserve capitalization:
**body → Body
attachment → Attachment
to → To
...**

I will let it like so for the time being.

Reply:
How about this?
```suggestion
- Send an email using the content of a Markdown file as body:
```

Reply:
Looks fine to me. Thanks. :)